handlers/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 - name: restart gitlab
   command: gitlab-ctl reconfigure
-  failed_when: false
```

Comment:
Still needs one newline at the end of the file. I'd also like to run a couple tests with https://github.com/geerlingguy/ansible-vagrant-examples/tree/master/gitlab to make sure this doesn't cause any problems there.